src/metabase/api/activity.clj

```diff
@@ -112,11 +114,34 @@
 (def ^:private views-limit 8)
 (def ^:private card-runs-limit 8)
 
-(api/defendpoint GET "/recent_views"
+(api/defendpoint ^:deprecated GET "/recent_views"
   "Get a list of 100 models (cards, models, tables, dashboards, and collections) that the current user has been viewing most
   recently. Return a maximum of 20 model of each, if they've looked at at least 20."
   []
-  {:recent_views (recent-views/get-list *current-user-id*)})
+  (u/snake-keys
+   (recent-views/get-recents *current-user-id* :views)))
```

Comment:
We can basically get rid of this endpoint, it's functionally the same as `GET activity/recents?context=views`.